components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/dto/APIThrottlingOverTimeDTO.java

```diff
@@ -0,0 +1,78 @@
+/*
+*  Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Use created year in the Copyright header rather a range. Check proper licence headers.